bokeh/models/glyphs.py

```diff
@@ -19,7 +20,19 @@ class Glyph(PlotObject):
 
     """
 
-    visible = Bool(help="""
+    def __init__(self, **kwargs):
+        props = dict()
+        if "line_alpha" in self.__class__.__dict__.keys():
+            props.update(themes.default['line_defaults'])
+        if "fill_alpha" in self.__class__.__dict__.keys():
+            props.update(themes.default['fill_defaults'])
+        if "text_alpha" in self.__class__.__dict__.keys():
+            props.update(themes.default['text_defaults'])
+        props.update(kwargs)
+        super(Glyph, self).__init__(**props)
```

Comment:
FYI not super happy with this, but it gets the job done for now. I am sure it can be improved.